code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -100,6 +100,9 @@
 	..()
 	new /obj/item/device/flashlight/seclite(src)
 
+/obj/item/weapon/storage/belt/security/empty/New()
+	..()
+

```

Comment:
Miauw, it does. The default secbelts New() spawns it with a seclite. if I don't put it here, every secbelt you make with the biogenerator would spawn with a seclite.